src/utils/useData.ts

```diff
@@ -1,4 +1,24 @@
-import fetch, { HeaderInit } from 'node-fetch';
+import fetch, {
+  HeaderInit,
+  RequestInfo,
+  RequestInit,
+  Response,
+} from 'node-fetch';
+import * as NodeCache from 'node-cache';
+
+import {
+  API_CACHE_ENTRIES_LIMIT,
+  API_CACHE_EXPIRATION_IN_SECONDS,
+  API_CACHE_REVALIDATION_WINDOW_IN_SECONDS,
+} from '../env';
+
+const apiCache = new NodeCache({
+  checkperiod: parseInt(API_CACHE_REVALIDATION_WINDOW_IN_SECONDS, 10),
+  stdTTL: parseInt(API_CACHE_EXPIRATION_IN_SECONDS, 10),
+  maxKeys: parseInt(API_CACHE_ENTRIES_LIMIT, 10),
```

Comment:
Should we even have this? If there's a rush of commands for the bot to search for things and this fills up to the limit, the cache probably starts throwing. See here:

https://github.com/node-cache/node-cache#options

On the other hand, if we're protected by the spam filter, this shouldn't be an actual problem, so cache entries should expire normally.